ompi/mca/coll/tuned/coll_tuned_reduce_decision.c

```diff
@@ -121,6 +124,15 @@ int ompi_coll_tuned_reduce_intra_check_forced_init (coll_tuned_force_algorithm_m
                                       MCA_BASE_VAR_SCOPE_ALL,
                                       &coll_tuned_reduce_chain_fanout);
 
+    coll_tuned_reduce_knomial_radix = 4;
```

Comment:
Already has a default value (line 34).

Reply:
These are two different variables for two different functions; one was intended for intranode gather/reduce; and the other one is for internode allgather/allreduce.  For internode allgather, we found out a large fanout (16) yield better results; and for intranode reduce, a fanout of 4 has better latency than the binomial reduce.

Reply:
No, they are not. Same name `coll_tuned_reduce_knomial_radix` in the same global scope. You might have intended something else, but that's not what this code does.